docs/spikes/2020-04-how-to-migrate-lb3-components.md

```diff
@@ -130,26 +130,25 @@ coming from._
 17. Add a new local service (a class providing JS/TS API), e.g. Push service or
     (database) Migration service.
 
-    - The service must be configurable by the target app (e.g. Apple & Google
-      credentials for push notifications, transport config for logger
-      frameworks, Amazon S3 credentials for storage component, etc.).
+    The service must be configurable by the target app (e.g. Apple & Google
+    credentials for push notifications, transport config for logger frameworks,
+    Amazon S3 credentials for storage component, etc.).
 
-      _LB3 typically uses component configuration. I think in LB4, we can use
-      `@configure` specific to each service (see the logger extension for an
-      example)._
+    _LB3 typically uses component configuration. I think in LB4, we can use
+    `@configure` specific to each service (see the logger extension for an
+    example)._
```

Comment:
Those two blocks are just whitespace changes, I removed the extra `-` bullet point.